airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/resources/github.py

```diff
@@ -36,6 +44,22 @@ def github_connectors_directory(resource_context: InitResourceContext) -> List[C
     return github_connector_repo.get_contents(connectors_path)
 
 
+@resource(
+    required_resource_keys={"github_connector_repo"},
+    config_schema={"connectors_path": StringSource},
+)
+def github_connectors_metadata_files(resource_context: InitResourceContext) -> List[str]:
+    resource_context.log.info(f"retrieving github metadata files")
+
+    github_connector_repo = resource_context.resources.github_connector_repo
```

Comment:
What's the type of this object? Are we interacting we a remote repo by getting raw file content with http requests to Github or are we cloning a the repo locally and read local files?

Reply:
Its the `Repository` type from `pygithub` https://github.com/airbytehq/airbyte/blob/bnchrch/stale-metadata-asset/airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/resources/github.py#L27

Which is all from the github api